### DISCUSSION TITLE
is there anyway to get past thiss? (blocked by crh

### DISCUSSION THREAD

**Discussion by plaqqq** [3kh0/ext-remover#608] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
![Screenshot 2023-04-14 4 34 36 PM](https://user-images.githubusercontent.com/128945462/232170700-f2b74215-8c0a-4c38-94a9-2ff6405dd97e.png)

**Comment by TheGooferFox** [comment, marked as answer]
I think an unblocked browser might do it but overall just use a L2PT VPN and u should be gud

**Reply by plaqqq** [reply]
i cant get vpn
